pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -188,8 +194,10 @@ type controller struct {
 	queue        workqueue.RateLimitingInterface
 	enqueueAfter func(*apisv1alpha1.APIBinding, time.Duration)
 
-	crdClusterClient apiextensionclientset.ClusterInterface
-	kcpClusterClient kcpclient.ClusterInterface
+	crdClusterClient                      apiextensionclientset.ClusterInterface
+	kcpClusterClient                      kcpclient.ClusterInterface
+	dynamicClusterClient                  dynamic.ClusterInterface
+	dynamicDiscoverySharedInformerFactory *informer.DynamicDiscoverySharedInformerFactory
```

Comment:
🔴  this is a problem: looking at enqueueAPIExport and enqueueAPIResourceSchema below, neither triggers claims to update, only the APIBindings they are used in directly.